mbed_lib.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "cmdline",
+  "config": {
+    "use_minumum_set": {
+      "help": "use pre configured minimum set which reduce most of features but allows to run icetea tests. Default: false",
```

Comment:
I'm not sure this makes it sufficiently clear that the flag defaults to false when this configuration is null or not set at all. 

Reading it right now the description says "default = false", but at the same time I can see that "value = null".

Would for example "Defaults to 'false' when not set" make sense?

Reply:
agree. That could be better. I'll change it a bit..